code/modules/mob/living/carbon/carbon.dm

```diff
@@ -362,7 +362,8 @@
 	if(HAS_TRAIT(src, TRAIT_SEE_IN_DARK))
 		see_in_dark = max(see_in_dark, 8)
 		lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
-
+	if(flags_pass & FLYING)	//Being up in the sky let's you see past things
+		sight |= SEE_THRU
```

Comment:
flags pass should not influence visuals also snowflake

Reply:
Why not? I can't add SEE_THRU and leave it at that since the proc is always running and removes the SEE_THRU.

Reply:
Because flags pass should be as the name implies for pass only